### PR TITLE
Fixes a few chems having incorrect sleep values, and fixes some flags on Ephed and Stimulants

### PR DESCRIPTION
**Pull request by Iamgoofball**
phil forgot to change some of these when doing 1tick

the ephedrine and stimulants were my fuckup, I forgot to switch them to GOTTAGOFAST when I renamed the old GOTTAGOFAST to IGNORESLOWDOWN

### CODE REVIEW

**Comment by phil235** [comment]
This should fix #9485

**Comment by PKPenguin321** [comment]
Oh, so virology stimulant *is* supposed to make you go super fast? I thought it making you ignore all speed debuffs was intended. :confused:

**Comment by RocKeD** [comment]
No it is supposed to speed you up, trust me viro is the job I play the most.

**Comment by PKPenguin321** [comment]
Yeah, I know it was originally like that, but I was like 99% sure at some point there was a *very intentional* change (I *think* by goof) that made it ignore slowdown instead.

**Comment by RocKeD** [comment]
That would make it completely useless, the only people that would benefit from it would be engineers if that were the case.

**Comment by PKPenguin321** [comment]
Yeah I found it it was intentionally changed by ye olde paprika
https://github.com/tgstation/-tg-station/pull/7721
>Stimulant virus will now simply remove any slowdown effects you have rather than making you 2fast2furious. Still, you can be hungry, freezing cold, near-death, and have 99 stamina damage, with a hardsuit on and you still won't move any slower than the default runspeed. It's still an immensely powerful virus.

It was intentional (but I have no problem with changing it back to the original function)

File: code/modules/reagents/Chemistry-Reagents/Medicine-Reagents.dm [hunk 1]
```diff
@@ -470,8 +470,8 @@ datum/reagent/medicine/morphine
 
 datum/reagent/medicine/morphine/on_mob_life(var/mob/living/M as mob)
 	M.status_flags |= IGNORESLOWDOWN
-	if(current_cycle >= 36)
-		M.sleeping += 3
+	if(current_cycle >= 12)
```
Comment (Shapsy) [inline code/modules/reagents/Chemistry-Reagents/Medicine-Reagents.dm]: what's the point of morphine if it knocks you out after 12 ticks

**Comment by GunHog** [comment]
I support the return of stimulant to its original purpose of MAKING YOU SANIC to achieve pre-HG levels. :+1:

File: code/modules/reagents/Chemistry-Reagents/Medicine-Reagents.dm (hunk 1 above)
Comment (Iamgoofball) [inline code/modules/reagents/Chemistry-Reagents/Medicine-Reagents.dm]: morphine was the replacement for sleep toxin actually

if/when I do pain and painkillers morphine will effect that too

**Comment by palpatine213** [comment]
Does returning it to the original gotta go fast let it still overcome the slowdown from space cold?

**Comment by Iamgoofball** [comment]
@palpatine213 Dunno, actually. It might, I'll need to check numbers.

**Comment by RocKeD** [comment]
@Iamgoofball I just played viro with stimulant and the speed does work again but now it has another bug, it doesn't ignore slowdown like it should.

**Comment by PKPenguin321** [comment]
I think it's not supposed to be both, just one or the other. Of course, that's really up to goof, but having both is kind of a huge buff.

**Pull request merged**